charts/karmada/README.md

```diff
@@ -207,9 +207,11 @@ helm install karmada-scheduler-estimator -n karmada-system ./charts/karmada
 | `kubectl.image.repository`               | Image of the kubectl                                                                                                                                                                                                                          | `"bitnami/kubectl"`                                                                                                                                                                                                  |
 | `kubectl.image.tag`                      | Image tag of the kubectl                                                                                                                                                                                                                      | `"latest"`                                                                                                                                                                                                           |
 | `kubectl.image.pullPolicy`               | Image pull policy of the kubectl                                                                                                                                                                                                              | `"Always"`                                                                                                                                                                                                           |
+| `kubectl.image.imagePullSecrets`               | Image pull secret of the kubectl                                                                                                                                                                                                              | `[]`                                                                                                                                                                                                           |
```

Comment:
`values.yaml` and `README.md` are not match:
`kubectl.image.imagePullSecrets` in `REAMDME.md` but `kubectl.image.pullSecrets` in `values.yaml`